openmc/plots.py

```diff
@@ -1296,30 +1374,128 @@ def from_xml_element(cls, elem):
             plot.wireframe_color = [int(item) for item in wireframe_color]
 
         # Set plot colors
-        colors = {}
-        xs = {}
+        plot._colors = {}
```

Comment:
Similar to the comment above, we can use the propert itself insteat of `_colors` here.

Reply:
And based on the `ProjectionPlot` constructor I don't think we need to reset these attributes here.

Reply:
ok!